### ISSUE TITLE
Current master is broken when configuring with zfp

### ISSUE THREAD

**Issue by williamfgc** [ornladios/ADIOS2#1742]
when using `-DADIOS2_USE_ZFP=ON` the following error shows up:

```
-- Verifying Fortran/CXX Compiler Compatibility - Success
CMake Error at testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:6 (include):
  include could not find load file:

    ADIOS2Functions
Call Stack (most recent call first):
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:165 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP3.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:33 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:168 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP3.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:33 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:171 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP3.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:33 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:165 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP4.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:61 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:168 (set_property):
  set_property given TEST names that do not exist:

    HeatTransfer.BP4.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:61 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


CMake Error at cmake/ADIOSFunctions.cmake:171 (set_property):
  set_property given TEST names that do not exist:

   HeatTransfer.BP4.zfp.Mx1.Validate

Call Stack (most recent call first):
  testing/examples/heatTransfer/TestBPFileMx1_zfp.cmake:61 (SetupTestPipeline)
  testing/examples/heatTransfer/CMakeLists.txt:13 (include)


-- Found MPI: TRUE (found version "3.1") found components:  C 

ADIOS2 build configuration:
  ADIOS Version: 2.4.0.442
  C++ Compiler : GNU 7.4.0 
    /usr/bin/c++

  Fortran Compiler : GNU 7.4.0 
    /usr/bin/f95

  Installation prefix: /home/wgodoy/tmp/adios2
        bin: bin
        lib: lib
    include: include
      cmake: lib/cmake/adios2
     python: lib/python3.6/site-packages

  Features:
    Library Type: shared
    Build Type:   Debug
    Testing: ON
    Build Options:
      Blosc    : ON
      BZip2    : ON
      ZFP      : ON
      SZ       : ON
      MGARD    : ON
      PNG      : ON
      MPI      : ON
      DataMan  : OFF
      Table    : OFF
      SSC      : OFF
      SST      : ON
      DataSpaces: OFF
      ZeroMQ   : ON
      HDF5     : ON
      Python   : ON
      Fortran  : ON
      SysVShMem: ON
      Profiling: ON
      Endian_Reverse: OFF
    RDMA Transport for Staging: Unconfigured

-- Configuring incomplete, errors occurred!
See also "/home/wgodoy/tmp/ADIOS2-build/CMakeFiles/CMakeOutput.log".
See also "/home/wgodoy/tmp/ADIOS2-build/CMakeFiles/CMakeError.log".
```

**Comment by williamfgc** [comment]
@chuckatkins I still see the issue on master and release branches with zfp. I am using Ubuntu 18.04

**Comment by chuckatkins** [comment]
Fixed by #1763
